README.md

```diff
@@ -1 +1,17 @@
 # TLA+ Foundation
+
+# 1. Goals of the Foundation
+The TLA+ Foundation (TLAF) is an independent, non-profit organization dedicated to fostering the adoption of the TLA+ specification language in industry, academia, and education, guided by the overall goal to advance the practice of mathematically thinking in software engineering.  The mission of TLAF is active support and funding of the development of the TLA+ tools to broaden their use by practitioners.  If not directly funding work, TLAF creates synergies whenever possible.
+
+The TLA+ Foundation is also responsible for the evolution of the TLA+ specification language that guarantees long-term consistency.
+
+# 2. The values of the Foundation
+How we pursue the goals of HF is just as important as what the goals are. These core principles will guide the actions of the Foundation:
+
+* **Open source**. The TLA+ foundation will embrace the open-source ethos wholeheartedly. Whenever TLAF develops or sponsors the development of tools and infrastructure, it will all be open source under a permissive license.  This is not limited to the end result but also applies to how work is organized and carried out; the community at large has to be able to participate at each step of the journey.
```

Comment:
```suggestion
* **Open source**. The TLAF will embrace the open-source ethos wholeheartedly. Whenever the TLAF develops or sponsors the development of tools and infrastructure, it will all be open-source. This is not limited to the end result but also applies to how work is organized and carried out; the community at large has to be able to participate at each step of the journey.
```
Do we have to commit to permissive licenses (MIT, BSD, etc. - not copyleft) in the charter? It can just be our operating practice without being codified.

Reply:
Committing to permissive licenses signals member companies long-term stability.

Why would we want a copyleft (GPL, ...) license in the first place?  From my experience, it makes things harder for most adopters.

Reply:
What does the rest of the group think about licenses?

Reply:
@lemmy I really just have a strong instinctive aversion to copyleft based on my experience working in open source, working with large bureaucratic companies that are adopting open source software, and speaking with attorneys specializing in open source software licensing. I think the question we’d need to answer first is: what benefit would we get from copyleft? I’m concerned it’s a risk without an obvious reward, but maybe someone here can justify it for us.

We’ve had discussions with attorneys on this topic at ONF in the past. If we think we need to investigate this further, I may be able to track down the recordings of those discussions and see if there’s any more helpful context in them.